### PR TITLE
Fix for installation script

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Review by Yao725 — APPROVED** [review]
LGTM

**Review by archerzz — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]
Should we do the same for all other RPs and the template?

**Comment by JoshLove-msft** [comment]
> Should we do the same for all other RPs and the template?

I have a PR that is doing this - https://github.com/Azure/azure-sdk-for-net/pull/28193
Which will hopefully be merging shortly 😄

File: sdk/grafana/Azure.ResourceManager.Grafana/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This package follows the [new Azure SDK guidelines](https://azure.github.io/azur
 Install the Azure Grafana management library for .NET with [NuGet](https://www.nuget.org/):
 
 ```PowerShell
```
Comment (JoshLove-msft) [inline sdk/grafana/Azure.ResourceManager.Grafana/README.md:11]: I'll update this to be ```dotnetcli in https://github.com/Azure/azure-sdk-for-net/pull/28193

**Pull request merged**